cargo-dist/src/linkage.rs

```diff
@@ -393,8 +393,13 @@ fn do_otool(path: &Utf8PathBuf) -> DistResult<Vec<String>> {
 fn do_ldd(path: &Utf8PathBuf) -> DistResult<Vec<String>> {
     let mut libraries = vec![];
 
+    // We ignore the status here because for whatever reason arm64 glibc ldd can decide
```

Comment:
strictly speaking, this should name ld as the primary culprit, because afaik a virtually-identical ldd script is run on both platforms (with the primary difference being in the default linker name selection... mine names some x86-specific options to try if `/usr/lib/ld-linux.so.2` doesn't work out), but it's the ld that ldd is wrapping that is going to cause it to provide amusingly different results.

Reply:
i am sad it caused a bug in our code but this stuff is so interesting to me. thanks for the additional detail!!